README.Rmd

```diff
@@ -38,3 +38,35 @@ library(comoDTC)
 ## basic example code
 ```
 
+## Contributing
+The comoDTC package is under active development. If you find a bug or have ideas about new features, please open an issue.
+
+If you've not created an R package before and would like to contribute, we'd recommend reading [the whole game](https://r-pkgs.org/whole-game.html) chapter of Wickham and Bryan.
+
+We ask that contributions to comoDTC follow the following workflow:
+
+1. create an issue that describes the change / addition to the code base you'd like to make
+2. create a new branch that is named `i[issue number]-[approximate issue name]`. For example, the "Create contributing docs issue" that was the basis for this change was issue #8, so the branch is named `i8-contributing-docs`
+3. run tests of code (see below) on your machine and check that they pass
+4. create a pull request which describes this contribution and refers back to the original issue
+5. check that pull request passes github workflow tests
+6. assign at least one reviewer
```

Comment:
Should we provide any suggestions for who they request as reviewers?

Reply:
I think it's ok to leave this open to them.